tests/commentHistory.test.js

```diff
@@ -0,0 +1,133 @@
+    chai
+      .request(server)
+      .post('/api/users/login')
+      .send({
+        email: 'alain2@gmail.com',
+        password: 'password23423'
+      })
+      .end((err, res) => {
+        userToken = res.body.user.token;
+        done();
+      });
+  });
+  it('Should check if the new comment is not a duplicate', (done) => {
+    chai
+      .request(server)
+      .patch('/api/articles/How-to-create-sequalize-seeds/comments')
+      .set('Authorization', userToken)
+      .send({
+        commentBody: 'Great work,canwe a part 2 of these article'
+      })
+      .end((err, res) => {
+        res.body.should.have.status(status.OK);
+        res.body.message.should.eq('The comment is not modifying what is already stored');
+        done();
+      });
+  });
+  it('Should reach the controller and create a new comment', (done) => {
+    chai
+      .request(server)
+      .patch('/api/articles/How-to-create-sequalize-seeds/comments')
+      .set('Authorization', userToken)
+      .send({
+        commentBody: 'Great work,can we a part 2 of these article'
+      })
+      .end((err, res) => {
+        res.body.should.have.status(status.CREATED);
+        res.body.comment.should.include({
+          id: 2,
+          articleSlug: 'How-to-create-sequalize-seeds',
+          username: 'BurindiAlain2',
+          commentBody: 'Great work,can we a part 2 of these article',
+          iteration: 1,
+        });
+        done();
+      });
+  });
+});
+
+describe('PATCH /api/articles/:slug/comments', () => {
+  it('Should create a new comment and iterate it by one', (done) => {
+    chai
+      .request(server)
+      .patch('/api/articles/How-to-create-sequalize-seeds/comments')
+      .set('Authorization', userToken)
+      .send({
+        commentBody: 'Great work,can we a part 3 of these article'
+      })
+      .end((err, res) => {
+        res.body.should.have.status(status.CREATED);
+        res.body.comment.should.include({
+          id: 3,
+          articleSlug: 'How-to-create-sequalize-seeds',
+          username: 'BurindiAlain2',
+          commentBody: 'Great work,can we a part 3 of these article',
+          iteration: 2,
+        });
+        done();
+      });
+  });
+});
+
+describe('GET /api/articles/:slug/comments/history', () => {
+  it('Should return comment history of the user on the particular article', (done) => {
+    chai
+      .request(server)
+      .get('/api/articles/How-to-create-sequalize-seeds/comments/history')
+      .set('Authorization', userToken)
+      .end((err, res) => {
+        res.body.should.have.status(status.OK);
+        res.body.comment.should.contain.something.like({
+          id: 3,
+          articleSlug: 'How-to-create-sequalize-seeds',
+          username: 'BurindiAlain2',
+          commentBody: 'Great work,can we a part 3 of these article',
+          iteration: 2,
+        });
+        done();
+      });
+  });
+  it('login and get token', (done) => {
```

Comment:
the same applies here

Reply:
https://github.com/andela/savage_rangers-ah/pull/69#discussion_r301181361

Reply:
can you please your names so that it can be less confusing while reviewing .It seems like the PR is for Alain more than you